README.md

```diff
@@ -19,19 +19,64 @@ source:
  - ruby-ronn
  - ruby-dev
 
+### Windows
+
+Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+Miniconda suffices.
+
+Create if necessary, and activate a Conda environment:
+
+```
+conda create -n ign-ws
+conda activate ign-ws
+```
+
+Install prerequisites:
+
+```
+conda install ruby git cmake --channel conda-forge
+```
+
 ## Installation
 
+### UNIX
+
 Standard installation can be performed in UNIX systems using the following
 steps:
 
- - mkdir build/
- - cd build/
- - cmake .. -DCMAKE_INSTALL_PREFIX=/usr
- - sudo make install
+```
+mkdir build/

```

Comment:
`build/` -> `build` and all below directories, just to keep consistency with the other tutorials